src/magnify/apps/core/models.py

```diff
@@ -15,3 +16,98 @@ class Meta:
 
     def __str__(self):
         return self.username
+
+class Meeting(models.Model):
+    """Model for one meeting or a collection of meetings defined reccursively"""
+    name = models.CharField(max_length=100)
+
+    # Start and end are the same for a single meeting
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
```

Comment:
since it's a datetime, don't call it date. I propose to just call them:
```suggestion
    start = models.DateTimeField()
    end = models.DateTimeField()
```